packages/wallet/.eslintrc.js

```diff
@@ -0,0 +1,241 @@
+  "@typescript-eslint/no-misused-new": "error",
+  "@typescript-eslint/no-namespace": "error",
+  "@typescript-eslint/no-parameter-properties": "off",
+  "@typescript-eslint/no-use-before-declare": "off",
+  "@typescript-eslint/no-var-requires": "error",
+  "@typescript-eslint/prefer-for-of": "off",
+  "@typescript-eslint/prefer-function-type": "error",
+  "@typescript-eslint/prefer-namespace-keyword": "error",
+  "@typescript-eslint/quotes": "off",
+  "@typescript-eslint/semi": ["off", null],
+  "@typescript-eslint/space-within-parens": ["off", "never"],
+  "@typescript-eslint/triple-slash-reference": "error",
+  "@typescript-eslint/type-annotation-spacing": "off",
+  "@typescript-eslint/unified-signatures": "error",
+  "@typescript-eslint/explicit-function-return-type": "off",
+  "@typescript-eslint/no-unused-vars": "off",
+  "@typescript-eslint/ban-ts-ignore": "off",
+  "@typescript-eslint/no-use-before-define": "off",
+  "@typescript-eslint/camelcase": "off"
+};
+
+module.exports = {
+  env: {
+    browser: true,
+    es6: true
+  },
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: "tsconfig.json",
+    ecmaVersion: 2018,
+    sourceType: "module"
+  },
+  plugins: ["@typescript-eslint", "@typescript-eslint/tslint", "prettier"],
+  extends: [
+    "plugin:@typescript-eslint/recommended",
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "plugin:import/typescript",
+    "plugin:react/recommended",
+    "prettier/@typescript-eslint",
+    "prettier"
+  ],
+  rules: Object.assign(rules, TSRules),
+  settings: {
+    "import/ignore": ["node_modules"],
+    react: {
+      version: "detect"
+    }
+  },
+  globals: {
+    // Maybe we should be importing these?
+    ethereum: "readonly",
+    web3: "readonly",
+    process: "readonly"
+  },
+  overrides: [
+    {
+      files: [
+        "*/**/{contract-tests,acceptance-tests,__tests__,__mocks__,__stories__}/**/*.{ts,tsx}"
+      ],
+      env: {
+        node: true
+      },
+      globals: {
+        jasmine: "readonly",
+        jest: "readonly",
+        Promise: "readonly",
+        expect: "readonly",
+        it: "readonly",
+        describe: "readonly",
+        beforeAll: "readonly",
+        beforeEach: "readonly",
+        fail: "readonly",
+        test: "readonly"
+      }
+    },
+    {
+      files: [
+        "src/setupTests.ts",
+        "bin/**/*.{js,ts}",
+        "config/**/*.{js,ts}",
+        "puppeteer/funding.ts"
+      ],
+      env: {
+        node: true
+      },
+      rules: {
+        "@typescript-eslint/no-var-requires": "off"
+      },
+      globals: {
+        jest: "readonly"
+      }
+    },
+    {
+      files: [".storybook/**/*.js", "{config,scripts}/**/*.js"],
+      parser: "babel-eslint",
+      env: {
+        node: true
+      },
+      rules: {
+        "@typescript-eslint/no-var-requires": "off"
+      },
+      globals: {
+        jest: "readonly"
+      }
+    }
+  ]
+};
```

Comment:
In general this config seems very long and complicated with many opinionated rules added that I don't think we have discussed as a team. I recommended trying to find ways to shorten it. For example, in the root `tslint.json` we currently extend some industry standard linting rules:

```js
  "extends": [
    "tslint:recommended",
    "tslint-plugin-prettier",
    "tslint-config-prettier"
  ]
```

Reply:
I used this to try and get the same rules:
https://github.com/typescript-eslint/tslint-to-eslint-config

I could just switch to using something similar to what you recommended but it might not have exactly the same settings.